integrationTests/rabbitmq/testNotifierWithRabbitMQ_test.go

```diff
@@ -98,41 +111,59 @@ func pushEventsRequest(webServer *integrationTests.TestWebServer, mutResponses *
 		BlockData: &outport.BlockData{
 			HeaderBytes: headerBytes,
 			HeaderType:  string(core.ShardHeaderV2),
-			HeaderHash:  []byte("headerHash"),
+			HeaderHash:  []byte("headerHash1"),
 			Body: &block.Body{
-				MiniBlocks: make([]*block.MiniBlock, 1),
+				MiniBlocks: []*block.MiniBlock{
+					&block.MiniBlock{},
+				},
 			},
 		},
 		TransactionPool:      txPool,
 		HeaderGasConsumption: &outport.HeaderGasConsumption{},
 	}
 
-	resp := webServer.PushEventsRequest(saveBlockData)
+	err := webServer.PushEventsRequest(saveBlockData)
+	log.LogIfError(err)
 
 	mutResponses.Lock()
-	responses[resp.Code]++
+	if err == nil {
+		responses[200]++
+	}
 	mutResponses.Unlock()
 }
 
-func pushRevertRequest(webServer *integrationTests.TestWebServer, mutResponses *sync.Mutex, responses map[int]int) {
-	blockEvents := &data.RevertBlock{
-		Hash:  "hash2",
-		Nonce: 1,
+func pushRevertRequest(webServer integrationTests.ObserverConnector, mutResponses *sync.Mutex, responses map[int]int) {
+	header := &block.HeaderV2{
+		Header: &block.Header{
+			Nonce: 1,
+		},
 	}
-	resp := webServer.RevertEventsRequest(blockEvents)
+	headerBytes, _ := json.Marshal(header)
+	blockData := &outport.BlockData{
+		HeaderBytes: headerBytes,
+		HeaderType:  string(core.ShardHeaderV2),
+		HeaderHash:  []byte("headerHash2"),
+	}
+	err := webServer.RevertEventsRequest(blockData)
+	log.LogIfError(err)
 
 	mutResponses.Lock()
```

Comment:
I think this lock/unlock could be moved inside if

Reply:
moved inside